packages/datadog-instrumentations/src/mysql.js

```diff
@@ -17,14 +17,14 @@ addHook({ name: 'mysql', file: 'lib/Connection.js', versions: ['>=2'] }, Connect
       return query.apply(this, arguments)
     }
 
-    const sql = arguments[0].sql ? arguments[0].sql : arguments[0]
+    const sql = arguments
```

Comment:
The magical `arguments` variable is a bit, well, magical! I think that while this code will work, passing `arguments` to other functions (in this case via `startCh.publish({ sql: sql, conf })`, so that it can be mutated elsewhere, is something that many people won't expect to happen.

I think an easier to understand version would be to do something like this:

```javascript
const query = arguments[0].sql ? arguments[0].sql : arguments[0]
const sql = { query }
startCh.publish({ sql, conf })
```

And then later where the published message is handled:

```javascript
sql.query = this.injectDbmQuery(sql.query)
```

That way, the mutation happens to a plain javascript object (`sql`), instead of to the magical `arguments` variable.